# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd .cargo/config.in
--- a/.cargo/config.in	Fri Jan 27 20:35:34 2023 +0000
+++ b/.cargo/config.in	Fri Jan 27 22:54:08 2023 +0200
@@ -102,7 +102,7 @@
 
 [source."https://github.com/mozilla/mp4parse-rust"]
 git = "https://github.com/mozilla/mp4parse-rust"
-rev = "eb0b625bd7e888d05ebcfc7685e2501b34c3b374"
+rev = "3bfc47d9a571d0842676043ba60716318e946c06"
 replace-with = "vendored-sources"
 
 [source."https://github.com/mozilla/neqo"]